keras/src/applications/mobilenet_v3.py

```diff
@@ -162,7 +162,11 @@ def MobileNetV3(
     dropout_rate=0.2,
     classifier_activation="softmax",
     include_preprocessing=True,
+    model_name=None,
```

Comment:
This is the base class, which isn't exported to the public API. For the argument to be configurable by the user, we need to add it to the functions that are exported to the public API (decorated with `@keras_export`), e.g. `def MobileNetV3Large`.

I suggest that we:

- Add a `name` argument to all public-facing `keras.applications` model-building functions. `name` is consistent with the layers API and the ops API.
- Make sure that by default it's the same name as what was previously hard-coded (e.g. `def VGG16(..., name="vgg16")`, to ensure backwards compatibility.

I will make a PR to do this for as many applications as I can manage -- can you do the MobileNet family on your side (including v1, v2, v3)?

Reply:
Sounds good! Done.